README.md

```diff
@@ -40,12 +49,12 @@ The tables lists the core parameters required to generate simulated data using m
 | local_dir| str: Parent directory on local machine to store intermediate results|
 | dataset_name| str: Name for analysis directory containing notebooks using ponyo|
 | raw_data_filename| str: File storing raw gene expression data|
-| normalized_data_filename| str: File storing normalized gene expression data|
+| normalized_data_filename| str: File storing normalized gene expression data. This file is generated by `normalize_expression_data()`.|
```

Comment:
It may be helpful to link to where the `normalize_expression_data` function lives